commands.py

```diff
@@ -0,0 +1,22 @@
+import os
+import json
+def create(schima):
+    parent_dir=os.getcwd()
+    data_base_name=schima["database_name"]
+    path=os.path.join(parent_dir,data_base_name)
+    os.mkdir(path)
+    for i in schima["Tables"]:
+        table_path=os.path.join(path,i["name"])
+        os.mkdir(table_path)
+        file_name=os.path.join(table_path,"info.txt")
+        f=open(file_name,'w')
```

Comment:
Did you. check if the dir exists? 
Why don't you have a private method call create folder instead of repeating this?